docs/attribute.md

```diff
@@ -0,0 +1,43 @@
+Annotation
+-------------------------
+You can use annotations for checking the feature state in controllers. Just active this in your config:
+
+```yml
+# config.yml
+
+flagception:
+    features:      
+        feature_123:
+            default: true
+        
+    # Use annotation? (optional)
+    annotation:
+    
+        # Enable controller annotation (default: false)
+        enable: true
+```
```

Comment:
This config including headline is about annotation and is irrelevant for attributes. Would you like to remove it?

Reply:
it's corrected !